### ISSUE TITLE
Reproducing the paper's method

### ISSUE THREAD

**Issue by xyf3626** [GameTechDev/TextureSetNeuralCompressionSample#10]
Thanks for your excellent work! When I try to reproduce the training method based on your paper, I encountered a problem . The compressed textures seem unable to reconstruct high-frequency details. Below are my training code, comparison images, and loss curve. I would greatly appreciate it if you could tell me what is going wrong.
In addition, I also want to know is there a warmup phase while you are training, and how do you initialize the BC1 parameters.

[training.py](https://github.com/user-attachments/files/28534100/training.py)

<img width="2921" height="1477" alt="Image" src="https://github.com/user-attachments/assets/7a875b12-5df2-468f-be49-ce92226b1232" />

<img width="2100" height="1050" alt="Image" src="https://github.com/user-attachments/assets/5aaecc34-8195-4733-8ed7-f4176420d7eb" />
